flake.nix

```diff
@@ -33,6 +33,11 @@
       ```
     */
 
+    nuschtosSearch = {
+      url = "github:NuschtOS/search";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
```

Comment:
As per other/prior feedback, can we get this from nixpkgs instead of a flake input?

Reply:
As mentionned, it does not have a nixpkgs package